aten/src/ATen/native/mps/operations/Copy.mm

```diff
@@ -290,8 +290,19 @@ void copy_blit_mps(void* dst, const void* src, size_t size) {
   if (dst.numel() == 0) {
     dst.resize_as_(src);
   }
+
+  TORCH_CHECK(dst.dim() >= src.dim());
   if (dst.dim() > src.dim()) {
     needs_broadcasting = true;
+  } else {
+    const IntArrayRef src_sizes = src.sizes();
```

Comment:
copy_ appears to have [broadcasting semantics](https://pytorch.org/docs/stable/notes/broadcasting.html)